core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,6 +47,47 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="icon-new-package icon-md" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
+      <l:task href="#modal" icon="icon-new-package icon-md" title="Remodal Popup"/>
+      <!-- <st:include page="popup.jelly" optional="true" /> -->
```

Comment:
So how can I include this popup.jelly ?
I tried to put popup.jelly to folders model, model/View, model/AllView but the error is the same. Header part of the form is inserted without errors, but a bigger part crashes.